realm/src/androidTest/java/io/realm/TestHelper.java

```diff
@@ -71,6 +77,22 @@ public static void copyRealmFromAssets(Context context, String realmPath, String
         is.close();
     }
 
+    // Creates a simple migration step in order to support null
+    // FIXME: generate a new encrypted.realm will null support
+    public static RealmMigration prepareMigrationStep() {
```

Comment:
Should be renamed to something like "prepareMigrationToNullSupportStep" or similar. The current name is way to generic.